soupsieve/css_match.py

```diff
@@ -956,9 +954,20 @@ def match_contains(self, el, contains):
                     content = self.get_text(el, no_iframe=self.is_html)
             found = False
             for text in contain_list.text:
-                if text in content:
-                    found = True
-                    break
+                if type(content) == str:
```

Comment:
I'd use `isinstance(content, str)`.

Reply:
`isinstance` here as well, or just a simple else as currently we only get a list or a string.

Reply:
Nice
I will change the code to follow your code style